test_util/test_upgrade_vpc.py

```diff
@@ -307,7 +307,9 @@ def setup_cluster_workload(self, dcos_api: DcosApiSession, healthcheck_app: dict
 
             dcos_api.marathon.deploy_app(viplisten_app)
             dcos_api.marathon.ensure_deployments_complete()
-            dcos_api.marathon.deploy_app(viptalk_app)
+            # viptalk app depends on VIP from viplisten app, which may still fail
```

Comment:
Yup!.  My comment was a mistake. I verified the apps and it's state are verified after the upgrade.

```
            def marathon_app_tasks_survive_upgrade():
            def test_mesos_task_state_remains_consistent():
```